plugins/experimental/ts_lua/ts_lua.c

```diff
@@ -166,7 +166,8 @@ TSRemapDeleteInstance(void *ih)
   int states = ((ts_lua_instance_conf *)ih)->states;
   ts_lua_del_module((ts_lua_instance_conf *)ih, ts_lua_main_ctx_array, states);
   ts_lua_del_instance(ih);
-  TSfree(ih);
+  // commenting it out - because now we reuse ts_lua_instance_conf for remap rules sharing the same lua script
+  // TSfree(ih);
```

Comment:
So now we will leak this? That doesn't seem reasonable... Where we've had similar problems, we end up ref-counting the instance data. I wish we could avoid that, and my thoughts was that we'd ref-count the entire remap config globally (per TXN or SSN), such that remap plugins never have to ref-count their instance data / continuations. But that's quite far out.

I'm not -1 on this, but at a minimum, if you do the above, you should change the comment such as 

1) Remove the TSfree()

2) Write a comment that you are now leaking this ih data on configuration reloads (assuming it does).